packages/go/analysis/analysis.go

```diff
@@ -240,10 +239,10 @@ func GetLAPSSyncers(tx graph.Transaction, domain *graph.Node) ([]*graph.Node, er
 	}
 }
 
-func GetDCSyncers(tx graph.Transaction, domain *graph.Node, filterTierZero bool) ([]*graph.Node, error) {
```

Comment:
This function is only called in one place so I opted for removing the parameter and filter logic instead of just passing false. I can add it back in and use the parameter if that is preferred, but I figured removing the unused code was cleaner.

Reply:
My vote is always to clean! 🧼